packages/cozy-harvest-lib/src/components/AccountModal.jsx

```diff
@@ -118,7 +118,7 @@ export class AccountModal extends Component {
     return (
       <>
         <KonnectorModalHeader konnector={konnector}>
-          {showAccountSelection ? (
+          {showAccountSelection && !konnector.clientSide ? (
```

Comment:
Can't we use the `showAccountSelection` props directly since it already does that same thing?

Reply:
@doubleface ?

Reply:
Right. Now i define this visibility in the upper level. I removed the unit tests